README.md

```diff
@@ -40,6 +37,9 @@
 - **Flexible and Expandable**: The modular design allows easy addition of new item types or enhancements to suit your needs.
 - **Efficient Search**: Search and retrieve items based on various criteria, enhancing the usability of your catalog.
 
+#### Demostration video <a name="live-demo"></a>
+  - [🚀 Watch the team talking about this project](https://drive.google.com/file/d/1J8v6inrGrJsUpXXhJ0QO91eOwAnqpa0l/view?usp=sharing)
```

Comment:
- [ ] Kindly keep your video with a maximum length of 5 minutes as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md#:~:text=It%20is%20a%20video%20with%20a%20maximum%20length%20of%205%20minutes.).

Reply:
Thank you. 💫💥
Readme updated with new Video link.
- do you Want to know what the authors are most proud of? 🚀 Watch the video, specially at 3:21 minute 👀.